### PR TITLE
{bio}[GCCcore/13.3.0] rDock v24.03.192

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/r/rDock/rDock-24.03.192-GCCcore-13.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+preinstallopts = 'export PREFIX=%(installdir)s && '
+
+runtest = 'test'
+
+postinstallcmds = ['sed -i "s|/bin/csh -f|/usr/bin/env tcsh -f|" %(installdir)s/bin/make_grid.csh']
+
+sanity_check_paths = {
+    'files': ['bin/rbhtfinder', 'bin/sdreport', 'bin/run_rbdock.pl', 'lib/libRbt.so'],
+    'dirs': ['include'],
+}
```
Comment (smoors) [inline easybuild/easyconfigs/r/rDock/rDock-24.03.192-GCCcore-13.3.0.eb:41]: can you add some `sanity_check_commands` here? for example
```python
sanity_check_commands = [
    'rbdock --help',
    'rbcavity --help',
]
```

Comment (sassy-crick) [reply easybuild/easyconfigs/r/rDock/rDock-24.03.192-GCCcore-13.3.0.eb:41]: Sorry for that, this has slipped through. Thanks for finding that.

**Comment by sassy-crick** [comment]
@boegelbot please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen3

**Review by smoors — APPROVED** [review]
lgtm

File: easybuild/easyconfigs/r/rDock/rDock-24.03.192-GCCcore-13.3.0.eb (hunk 1 above)
Comment (smoors) [reply easybuild/easyconfigs/r/rDock/rDock-24.03.192-GCCcore-13.3.0.eb:41]: no worries, no need to apologize :)

**Comment by smoors** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**